templates/TemplateNotebook_HatchetSpotDB.ipynb

```diff
@@ -0,0 +1,142 @@
+{
```

Comment:
I think we'll want to clear out the cells so users get an empty template notebook.

Reply:
Yeah good point, I deleted the output